docs/markdown/Introduction/media.md

```diff
@@ -4,10 +4,94 @@ slug: "media"
 excerpt: "Learn more about Pants and related topics from these talks, posts and podcasts featuring Pants contributors and users."
 hidden: false
 createdAt: "2021-04-18T17:27:18.361Z"
-updatedAt: "2022-04-05T22:02:38.457Z"
+updatedAt: "2022-09-08T22:02:38.457Z"
 ---
-Talks
------
+## Posts & Articles
+
+### Coinbase Blog
+
+September 1, 2022  
+[**Case Study**] "Building a Python ecosystem for efficient and reliable development"  
+<https://blog.coinbase.com/building-a-python-ecosystem-for-efficient-and-reliable-development-d986c97a94a0>
+
+> Part 1 of 2. Describes how the company developed an efficient, reliable Python ecosystem using Pants, and solved the challenge of managing Python applications at a large scale at Coinbase.
+
+### IBM Developer Blog
+
+August 24, 2022  
+[**Case Study**] "Incrementally migrating a Python monorepo from Bazel to Pants"  
+<https://developer.ibm.com/blogs/case-study-incrementally-migrating-a-python-monorepo-from-bazel-to-pants/>  
+
+> Watson Orders is an IBM Silicon Valley based technology development group targeting the development of world-class conversational AI. This posts walks through the process of migrating off Bazel, where they maintained 19,000 lines of BUILD file metadata, to Pants where that was slashed to 2,400 lines thanks to [dependency inference](doc:/how-does-pants-work#dependency-inference). CI build time for PRs dropped from 10-12 minutes with Bazel to under 4 minutes with Pants.
+
+### Astranis Space Technologies
+
+August 12, 2022
+[**Case Study**] "Astranis Case Study: Wrangling Python In a Monorepo"
```

Comment:
This may be a difference between GH-flavored Markdown vs whatever flavor Readme's parser uses, because it renders a linebreak with the same edit: https://www.pantsbuild.org/v2.12/edit/media

But I've pushed a revision with the space restored, for the sake of consistency and review. That there is discrepancy between how GH and Readme renders Markdown is unfortunate.